src/apps_plugin.c

```diff
@@ -725,21 +727,30 @@ long all_pids_count = 0;
 
 struct pid_stat *get_pid_entry(pid_t pid)
 {
+	// FIXME: The double linked list pointed by root_of_pids
+	//        is garanteed to be a sorted list?!
+
```

Comment:
no! this is why the is the array `all_pids` that links the proper `pid_stat` to its id. So we have random access on the linked list. This is also why we needed the max pid of the system: the plugin allocates `all_pids` for all possible pids.